### ISSUE TITLE
Ask about the results of paper reproduction

### ISSUE THREAD

**Issue by daichuangye** [HUANGLIZI/LViT#37]
Hello!! Thank you very much for your work, LViT paper is very helpful to me!!

Here I would like to ask about my results of replicating QaTa-COV19 data set on LViT model.

I used the QaTa-COV19 data set provided by you. The training set is 5716 pieces and the verification set is 1429 pieces. The division of the training set and the verification set according to the text.xlsx file you provided.

The Batch_size is set to 24, the learning rate is set to 3e-4, the learning rate is reduced to 1e-4 minimum after 50 epochs, if the effect does not improve, then stop training

I didn't make any changes to the original code I downloaded, and the random torrent uses 666 from the original code

However, the Dice value I reproduced was 81.97, which was slightly different from the result recorded in the paper

The device I am using is RTX 3090 24GB video memory and the Python version is 3.7.2
I have sent you an email and I am looking forward to your reply！！

**Comment by HUANGLIZI** [comment]
Unfortunately, I am unable to provide an explanation for the reproduction results under other specific conditions. You may need to load pretrained UNet models or increase the number of early-stop epochs if you want to get better results.
